provisioner/ansible/provisioner.go

```diff
@@ -805,6 +805,15 @@ func (p *Provisioner) executeAnsible(ui packer.Ui, comm packer.Communicator, pri
 	return nil
 }
 
+func cleanUpAnsibleDir(ui packer.Ui) {
+	ansibleDir := "~/.ansible"
```

Comment:
Here `~` might not mean something on all systems ( like windows ). To set `ansibleDir` we could :

```suggestion
	usr, err := user.Current()
	if err != nil {
		 ui.Error(fmt.Sprintf("Failed to clean up: failed to get current user: %v", err))
	}
	ansibleDir := filepath.Join(usr.HomeDir, "ansible")
```
( we probably will have to import those )

Reply:
Thanks @azr , fixed this accordingly 👍

Reply:
nice, thanks !

Reply:
No @azr, I saw there was an issue raised as "good-first-issue" and picked it up. Maybe @SwampDragons can help in testing this? Or let me know spend sometime to test this out.